README.md

```diff
@@ -15,8 +15,9 @@ This repo provides various form of distribution for tezos-related executables
 Daemon binaries (as well as packages for them) have suffix that defines their target protocol,
 e.g. `tezos-baker-007-PsDELPH1` can be used only on the chain with 007 protocol.
 
-Other binaries can be used with all protocols. E.g. `tezos-node` can be set up to run different
-networks, more about this you can read in [this article](https://tezos.gitlab.io/user/multinetwork.html).
+Other binaries can be used with all protocols if they're new enough. E.g.
+007 protocol is supported only from `v7.4`. `tezos-node` can be set up to run
+different networks, more about this you can read in [this article](https://tezos.gitlab.io/user/multinetwork.html).
```

Comment:
Is it grammatically correct btw? Isn't "you can read more about this" more :proper:?